sites/partners/__tests__/pages/applications/index.test.tsx

```diff
@@ -0,0 +1,243 @@
+    expect(getByText("Last Name HH:1")).toBeInTheDocument()
+    expect(getByText("Household Last")).toBeInTheDocument()
+    expect(getByText("Household DOB HH:1")).toBeInTheDocument()
+    expect(getByText("11/25/1966")).toBeInTheDocument()
+    expect(getByText("Relationship HH:1")).toBeInTheDocument()
+    expect(getByText("Friend")).toBeInTheDocument()
+    expect(getByText("Same Address as Primary HH:1")).toBeInTheDocument()
+    expect(getByText("Work in Region HH:1")).toBeInTheDocument()
+    expect(getAllByText("Yes")).toHaveLength(3)
+    expect(getByText("Flagged as Duplicate")).toBeInTheDocument()
+    expect(getByText("Marked as Duplicate")).toBeInTheDocument()
+    expect(getByText("No")).toBeInTheDocument()
+  })
+
+  it("should directly open application add page when add application is clicked while listing is open", async () => {
+    const { pushMock } = mockNextRouter({ id: "Uvbk5qurpB2WI9V6WnNdH" })
+
+    server.use(
+      rest.get("http://localhost/api/adapter/listings/Uvbk5qurpB2WI9V6WnNdH", (_req, res, ctx) => {
+        return res(ctx.json(listing))
+      }),
+      rest.get("http://localhost:3100/applications", (_req, res, ctx) => {
+        return res(ctx.json({ items: [application], meta: { totalItems: 1, totalPages: 1 } }))
+      }),
+      rest.get("http://localhost/api/adapter/applications", (_req, res, ctx) => {
+        return res(ctx.json({ items: [application], meta: { totalItems: 1, totalPages: 1 } }))
+      }),
+      rest.get("http://localhost:3100/user/list", (_req, res, ctx) => {
+        return res(ctx.json({ items: [user], meta: { totalItems: 1, totalPages: 1 } }))
+      }),
+      rest.get("http://localhost/api/adapter/user/list", (_req, res, ctx) => {
+        return res(ctx.json({ items: [user], meta: { totalItems: 1, totalPages: 1 } }))
+      }),
+      rest.get("http://localhost/api/adapter/applicationFlaggedSets", (_req, res, ctx) => {
+        return res(ctx.json({ items: [], meta: { totalItems: 0, totalPages: 0 } }))
+      }),
+      rest.get("http://localhost/api/adapter/applicationFlaggedSets/meta", (_req, res, ctx) => {
+        return res(ctx.json({ totalCount: 1 }))
+      }),
+      // set logged in user as admin
+      rest.get("http://localhost/api/adapter/user", (_req, res, ctx) => {
+        return res(ctx.json({ id: "user1", roles: { id: "user1", isAdmin: true } }))
+      })
+    )
+    const { findByText, getByText } = render(<ApplicationsList />)
+
+    const header = await findByText("Partners Portal")
+    expect(header).toBeInTheDocument()
+
+    fireEvent.click(getByText("Add Application"))
+    expect(pushMock).toHaveBeenCalledWith("/listings/Uvbk5qurpB2WI9V6WnNdH/applications/add")
+  })
+
+  // This is blocked by a Seeds PR to update the Dialog configuration
+  it.skip("should open confirmation modal when application add is clicked while listing is closed", async () => {
+    const { pushMock } = mockNextRouter({ id: "Uvbk5qurpB2WI9V6WnNdH" })
```

Comment:
This test is what is testing the new behavior, the rest is setup and other test coverage for this file